File: Modul4/Soal 4/PRAK304-2210817210010-Eric Nanda Ferdian.py
a = int(input(''))

if a >=1 and a <=9:
    b = ('Satuan')
elif a == 0:
    b = ('Nol')
elif a > 99:
    b = ('Anda Menginput Melebihi Limit Bilangan')
elif a >=20 and a <=99:
    b = ('Puluhan')
elif a >=11 and a <=19:
    b = ('Belasan')
print('%s'% b)

File: Modul4/Soal 4/PRAK304-2210817210010-Eric Nanda Ferdian.c
#include <stdio.h>

int main()
{
    int a;
    scanf("%d",&a);
    if(a>=1&&a<=9)
    {
        printf("Satuan");
    }
    else if(a==0)
    {
        printf("Nol");
    }
    else if(a>99)
    {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    else if(a>=20&&a<=99)
    {
        printf("Puluhan");
    }
    else if(a>=11&&a<=19)
    {
        printf("Belasan");
    }
}
